### PR TITLE
Add Dragon Quest® VIII Journey of the Cursed King

### PR DESCRIPTION
**Pull request by LCypher00**
Adds Dragon Quest® VIII Journey of the Cursed King.

*Automatically generated by PR builder*

### CODE REVIEW

**Comment by LCypher00** [comment]
Just created the wiki page for the game (albeit a bit lacking :/ )

**Comment by Schplee** [comment]
LGTM - Just need a second reviewer to sign off.

File: games/dragon-quest-viii-journey-of-the-cursed-king/game.dat [hunk 1]
```diff
@@ -0,0 +1,29 @@
+title = "Dragon Quest® VIII Journey of the Cursed King"
+description = "Published by Square Enix, Dragon Quest® VIII Journey of the Cursed King is a remake of the original game for PS2. It's a JRPG set in a world full of weird looking monsters where you take control of an unnamed hero seeking to lift the curse cast by the evil Dhoulmagus, off of king Trode and the princess Medea ,who both  have been  transformed respectively into a monster and a horse, and the castle of Trodain."
```
Comment (j-selby) [inline games/dragon-quest-viii-journey-of-the-cursed-king/game.dat]: Spacing at the end of this description is off

File: games/dragon-quest-viii-journey-of-the-cursed-king/game.dat [hunk 2]
```diff
@@ -0,0 +1,29 @@
+title = "Dragon Quest® VIII Journey of the Cursed King"
+description = "Published by Square Enix, Dragon Quest® VIII Journey of the Cursed King is a remake of the original game for PS2. It's a JRPG set in a world full of weird looking monsters where you take control of an unnamed hero seeking to lift the curse cast by the evil Dhoulmagus, off of king Trode and the princess Medea ,who both have been transformed respectively into a monster and a horse, and the castle of Trodain."
```
Comment (Schplee) [inline games/dragon-quest-viii-journey-of-the-cursed-king/game.dat]: `princess Medea ,who both have` - Comma is misplaced

**Pull request merged**